soknad-pleiepenger-barn/src/main/java/no/nav/k9/søknad/pleiepengerbarn/PleiepengerBarnSøknadValidator.java

```diff
@@ -138,6 +140,12 @@ private void validerArbeid(Arbeid arbeid, List<Feil> feil) {
                 if (skalJobbeProsent == null || skalJobbeProsent.doubleValue() < 0 || skalJobbeProsent.doubleValue() > 100) {
                     feil.add(new Feil("arbeid.arbeidstaker[" + i + "].perioder[" + perioder.getKey().iso8601 + "].skalJobbeProsent", "ugylidigProsent", "Skal jobbe prosent må være mellom 0 og 100"));
                 }
+                Duration jobberNormalPerUke = perioder.getValue().jobberNormaltPerUke;
+                if (jobberNormalPerUke != null && jobberNormalPerUke.isNegative()) {
+                    feil.add(new Feil("arbeid.arbeidstaker[" + i + "].perioder[" + perioder.getKey().iso8601 + "].jobberNormalPerUke", "ugyldigJobbperiode", "Jobber normalt per uke må settes til en gyldig verdi."));
+                } else if (jobberNormalPerUke != null && jobberNormalPerUke.compareTo(MAKS_INNENFOR_EN_UKE) > 0) {
+                    feil.add(new Feil("arbeid.arbeidstaker[" + i + "].perioder[" + perioder.getKey().iso8601 + "].jobberNormalPerUke", "ugyldigJobbperiode", "Jobber normalt per uke kan ikke overstige en uke."));
+                }
```

Comment:
Hvis `jobberNormalPerUke` er et påkrevd felt for behandling i k9-sak bør validatoren kreve dette feltet. Dette fordi både brukerdialog og k9-punsj da må levere en oppdatert versjon.

Gamle versjoner ville kunne rutes til Gosys med k9-fordel basert på versjonsnummer (selv om vi ikke trenger å gjøre dette nå grunnet at vi kun kjører i test). Eller bedre: Gosysoppgave fordi den ikke validerer :-)

Reply:
Ja sant det beste er jo om den er required. Tenkte da jeg lagde dette at vi allerede opprettet oppgaver for pleiepenger i prod, men det er jo bare for omsorgspenger enda.

Så kan fikse det, og da er det egentlig ikke noe behov fort den Utils-metoden heller

Reply:
Fint hvis du kan følge opp at endringene blir gjort riktig i brukerdialog og k9-punsj :-)